docker-compose.yml

```diff
@@ -48,8 +48,24 @@ services:
       - neo4j-logs:/logs
       - neo4j-import:/var/lib/neo4j/import
       - neo4j-plugins:/plugins
+  postgres:
+    container_name: mcs-postgres
+    environment:
+      POSTGRES_USER: "mcs"
+      POSTGRES_PASSWORD: "7EAdu7jJvZNxxrNZ"
+      POSTGRES_DB: "kg"
```

Comment:
This is why you don't need the table name prefixes. neo4j only supports multiple databases in the enterprise versions.

Reply:
Ah interesting